controllers/users/users.js

```diff
@@ -12,24 +12,26 @@ const sendVerificationEmail = user => {
     service: "gmail",
     auth: {
       user: process.env.GMAIL_USER,
-      pass: process.env.GMAIL_PASS,
+      pass: process.env.GMAIL_PASS
     }
-  })
+  });
 
   const mailOptions = {
     from: "natripareact <natripareact@gmail.com>",
     to: user.email,
     subject: "Account Activation",
     html: `
       To activate your account, click this link
-      <a href="http://localhost:5000/api/users/activate/${user.activationKey}">Activate Account</a>
+      <a href="http://localhost:5000/api/users/activate/${
+        user.activationKey
+      }">Activate Account</a>
```

Comment:
Completely unreadable for me if there will be more variables in template literals